lib/bloc/fee_options/fee_options_bloc.dart

```diff
@@ -3,31 +3,37 @@ import 'dart:async';
 import 'package:breez_sdk/breez_bridge.dart';
 import 'package:breez_sdk/bridge_generated.dart';
 import 'package:breez_translations/breez_translations_locales.dart';
-import 'package:c_breez/bloc/withdraw/withdraw_funds_state.dart';
+import 'package:c_breez/bloc/fee_options/fee_option.dart';
+import 'package:c_breez/bloc/fee_options/fee_options_state.dart';
 import 'package:c_breez/utils/exceptions.dart';
 import 'package:fimber/fimber.dart';
 import 'package:hydrated_bloc/hydrated_bloc.dart';
 
-final _log = FimberLog("WithdrawFundsBloc");
+final _log = FimberLog("FeeOptionsBloc");
 
-class WithdrawFundsBloc extends Cubit<WithdrawFundsState> {
+class FeeOptionsBloc extends Cubit<FeeOptionsState> {
   final BreezSDK _breezLib;
 
-  WithdrawFundsBloc(
-    this._breezLib,
-  ) : super(WithdrawFundsState.initial());
+  FeeOptionsBloc(this._breezLib) : super(FeeOptionsState.initial());
 
-  Future sweep({
-    required String toAddress,
-    required int feeRateSatsPerVbyte,
-  }) async {
-    _log.v("Sweep to address $toAddress using $feeRateSatsPerVbyte fee vByte");
-    await _breezLib.sweep(
-      toAddress: toAddress,
-      feeRateSatsPerVbyte: feeRateSatsPerVbyte,
-    );
+  /// Lookup the most recent reverse swap pair info using the Boltz API
+  Future<ReverseSwapPairInfo> fetchReverseSwapFees({int? amountSat}) async {
```

Comment:
I think this function doesn't belong here. Maybe belongs to the reverse swap bloc?